src/pyodide.js

```diff
@@ -432,6 +432,15 @@ globalThis.loadPyodide = async function(config = {}) {
       false; // we preload wasm using the built in plugin now
   Module.preloadedWasm = {};
 
+  /**
+   * Tell Pyodide about an available copy of Comlink.
+   * Necessary to enable use of Comlink Proxies from Python.
+   * @private
+   */
+  Module.registerComlink = function(Comlink){
+    Module.Comlink = Comlink;
```

Comment:
Something like
```suggestion
    Module._Comlink = Comlink;
```
might be a bit more explicit about the fact that this object is not part of the public API.